README.md

```diff
@@ -183,8 +183,12 @@ OPTIONS
     --password <string>, $GIT_SYNC_PASSWORD
             The password or personal access token (see github docs) to use for
             git authentication (see --username).  NOTE: for security reasons,
-            users should prefer the environment variable for specifying the
-            password.
+            users should prefer using file for specifying the password (see
```

Comment:
Super not: " using a file " - missing "a" (yay English).

Reply:
Updated.